cmd/restore.go

```diff
@@ -110,8 +112,13 @@ func runRestore(flagSet *flag.FlagSet, cmdName, dbName, tableName string) error
 		}
 	case len(dbName) != 0 && len(tableName) == 0:
 		// database restore
+		fmt.Fprintf(os.Stderr, "dbName: %s\n", dbName)
 		db := client.GetDatabase(dbName)
-		err = client.CreateDatabase(db.Schema)
+		if db == nil {
+			err = errors.Errorf("database %s not found in backup", dbName)
```

Comment:
This is re-using and `err` variable from above, which is considered brittle. In this case you can just immediately return the error without assigning to a variable. And you don't even need `errors.Trace` because `errors.Errorf` already ensures a stack trace is created.